docs/src/main/sphinx/release/release-398.md

```diff
@@ -0,0 +1,64 @@
+# Release 398 (28 Sep 2022)
+
+## General
+
+* Add Hudi connector. ({issue}`10228`)
+* Add metrics for the execution time of filters and projections to `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`14135`)
+* Show local cost estimates when using `EXPLAIN`. ({issue}`14268`)
+* Fix timeouts happening too early because of improper handling of the
+  `node-scheduler.allowed-no-matching-node-period` configuration property. ({issue}`14256`)
+* Fix query failure for `MERGE` queries when `task_writer_count` is greater
+  than one. ({issue}`14306`)
+
+## CLI
+
+* Add display for the results of `EXPLAIN ANALYZE`. ({issue}`13907`)
+
+## Accumulo connector
+
+* Add support for column comments when creating a new table. ({issue}`14114`)
+* Add column mapping and index information in an extra field instead of a
+  comment. ({issue}`14095`)
+
+## BigQuery connector
+
+* Fix SQL injection vulnerability. ({issue}`14254`)
+* Fix query failure when the predicate involves a varchar value with a `\`. ({issue}`14254`)
+
+## ClickHouse connector
+
+* Upgrade minimum required version to 21.8. ({issue}`14112`)
```

Comment:
Does that bring some feature or benefit that we can add?

Reply:
No, but we've included this in release notes before, so I figured it was worth including again.

Reply:
Yeah . just wondering .. would be good to add some benefit ;-)